### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by evmorov** [michaelvillar/timer-app#12]
The last release was on May 18, 2016. In `homebrew cask` there is the same version 1.1. This version doesn't have 'stay on top' feature.

Can you release a new version or provide an instruction in the `README.md` how to build the app from source?

**Comment by krokofant** [comment]
I am not familiar with Xcode but I've managed to build it by
* Cloning the project
* Opening the project in Xcode
* Removing everything that mentions signing, especially the *Team* at the screen below 
![image](https://cloud.githubusercontent.com/assets/5908498/22865732/7493fff4-f16a-11e6-9773-6fe60c085181.png)
* cd'ing into the project folder and running:
```
xcodebuild SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk
```

I previously got an error saying `<unknown>:0: error: Swift does not support the SDK 'MacOSX10.11.sdk'` with the SDKROOT for *MacOSX10.11.sdk* so it seems I had to override it. I have no idea why.

But now it works like a charm. Cool app!
